core/src/main/java/io/grpc/internal/Stream.java

```diff
@@ -79,6 +79,9 @@
    */
   boolean isReady();
 
+
+  void start();
```

Comment:
I would have expected ClientStreamListener to be passed in, although I guess this works.